Vic2ToHoI4/Data_Files/configurables/regions.txt

```diff
@@ -1,1489 +1,2252 @@
-	}
-}
-
-aotearoa = {
-	name = "Aotearoa"
-	adjective = "Aotearoan"
+		# Iceland & Greenland
+		10245 10636 12512 13058 13066 4861 12674 12689 
+
+		# Faroe
+		13003
+		
+		# Nordnorge
+		21 38 64 76 91 107 115 128 173 188 192 2995 3016 3018 3022 3034 3040 3048 3058 3082 3088 3096 3098 3126 3132 3145 3180 6015 6038 6041 6071 6079 6092 6107 6111 6116 6118 6130 6173 6187 6214 9042 9064 9069 9089 9116 9126 9127 9128 9132 9139 9191 9208 9216 11034 11050 11063 11077 11100 11104 11118 11169 11173 22 44 11090 11157 
+
+		# Vestlandet
+		14 53 65 100 122 137 199 3049 3059 3090 3094 3114 3129 3188 3192 6072 6080 6145 6151 6176 6184 6216 9036 9047 9096 9170 9174 9196 9204 9296 11029 11101 11176 11198 11367 
+
+		# Ostlandet
+		10 27 102 111 133 144 179 3007 3021 3093 3105 3136 3171 3183 6042 6114 6115 6136 6160 6204 9032 9103 9135 9157 9227 11028 11037 11078 11113 11135 11151 11184 11206 
+
+		# Norrland
+		11 46 49 58 61 101 109 113 118 125 130 145 148 151 168 182 184 190 201 202 205 3003 3039 3044 3045 3054 3062 3095 3100 3104 3108 3111 3113 3138 3162 3173 3179 3191 6010 6023 6024 6055 6063 6067 6074 6083 6125 6126 6129 6132 6152 6155 6161 6165 6168 6170 6185 6200 6206 6207 6208 9035 9051 9066 9083 9088 9093 9095 9108 9117 9134 9141 9143 9147 9151 9154 9171 9175 9178 9186 9211 9220 9226 9229 11026 11040 11049 11052 11058 11060 11067 11069 11073 11091 11102 11112 11116 11120 11123 11129 11147 11148 11152 11155 11185 11187 11190 11201 11208 11209 
+
+		# Svealand
+		35 69 77 114 147 156 161 3005 3070 3106 3128 3141 3148 3182 6028 6050 6084 6093 6123 6127 6147 6164 6188 6195 6209 6311 6397 9054 9068 9109 9138 9148 9149 9173 9182 9185 9193 9209 9218 9308 9393 11044 11053 11070 11082 11114 11163 11188 11197 11278 11289 11380 
+
+		# Götaland
+		171 211 242 295 299 329 363 383 3063 3229 3286 3307 3375 3386 6120 6148 6310 6331 6345 6406 6412 9251 9279 9300 9329 9362 9406 9410 11160 11215 11231 11280 11311 11327 11376 11390 408 6258
+
+		# Northern Finland
+		7 30 37 52 82 103 120 124 132 136 139 141 158 162 165 167 193 195 197 203 2992 3013 3026 3030 3055 3068 3069 3073 3097 3107 3115 3125 3131 3149 3153 3156 3158 3185 3193 6012 6033 6045 6061 6076 6078 6081 6087 6128 6134 6137 6142 6146 6156 6179 6183 6210 6212 9056 9061 9073 9104 9106 9112 9140 9163 9169 9181 9189 9223 9233 11024 11030 11041 11043 11076 11079 11081 11085 11115 11121 11125 11130 11132 11138 11141 11142 11164 11167 11199 11204 11210 
+
+		# Southern Finland
+		32 63 66 67 169 204 3009 3025 3050 3060 3061 3083 3117 3119 3122 3133 3137 3161 3172 3187 6009 6013 6047 6052 6066 6082 6108 6138 6141 6144 6150 6153 6163 6186 6198 9038 9094 9100 9105 9129 9150 9156 9160 9166 9167 9176 9183 9194 9206 9219 11033 11046 11056 11065 11066 11105 11126 11144 11156 11162 11172 11211
+
+		# Åland Islands
+		9228
+
+		# Jylland
+		316 394 399 3206 3277 3341 6235 6364  
+
+		# Sjaelland
+		3260 3305 3325 3374 6287 6361 6393 
+	}
+}
+baltic = {
```

Comment:
I'm not really sold on how fine-grained many of these are. They seem pretty based on OTL countries rather than geography or larger regions.

Reply:
They are based on IRL countries, but of course the splitting of Europe in regions can be made more topographically based in a later PR. It is just a step higher from our current UN regions division because it makes the 1930s colonies setup possible.